services/app/src/components/views/ExpansionCheckbox.tsx

```diff
@@ -0,0 +1,108 @@
+import Checkbox from '@material-ui/core/Checkbox';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormGroup from '@material-ui/core/FormGroup';
+import ToolTip from '@material-ui/core/ToolTip';
+import * as React from 'react';
+import { ContentSetsType } from '../../reducers/StateTypes';
+
+interface IExpansion {
+  checked: boolean;
+  value: string;
+  disabled: boolean;
+  label: string;
+}
+
+export interface IState {
+  expansions: IExpansion[];
+}
+
+export interface IProps {
+  contentSets: ContentSetsType;
+  onChange: (values: string[]) => void;
+  value: string[] | undefined;
+}
+
+export class ExpansionCheckbox extends React.Component<IProps, {}> {
+  public state: IState;
+  constructor(props: IProps) {
+    super(props);
+    const isHorror = this.props.contentSets.horror ? true : false;
+    const isFuture = this.props.contentSets.future ? true : false;
```

Comment:
Shouldn't you be able to use content sets.future directly as a Boolean?

Reply:
So by default `future` is undefined.